betterconf/caster.py

```diff
@@ -11,15 +11,18 @@ def cast(self, val: str):
 
 class ConstantCaster(AbstractCaster, typing.Generic[VT]):
 
-    ABLE_TO_CAST: typing.Dict[str, typing.Any] = {}
+    ABLE_TO_CAST: typing.Dict[typing.Union[str, typing.Tuple[str]], typing.Any] = {}
 
     def cast(self, val: str) -> typing.Union[str, VT]:
         """Cast using ABLE_TO_CAST dictionary as in BoolCaster"""
-        converted = self.ABLE_TO_CAST.get(val.lower())
-        if not converted:
-            return val
-        else:
+        if val in self.ABLE_TO_CAST.keys():
```

Comment:
I think we should use just search by `hash`, not in just `.keys()` list.

Reply:
> I think we should use just search by `hash`, not in just `.keys()` list.

What do you mean by that? Everything I found is about internal Python dict lookup...

Reply:
> > I think we should use just search by `hash`, not in just `.keys()` list.
> 
> What do you mean by that? Everything I found is about internal Python dict lookup...

Oh, sure. I meant that searching with `something in dict` is faster than `something in dict.keys()`, because internally second way is like searching in plain list.